Sheets node name (#126)
datacommonsorg/docsite: api/sheets/get_name.md

grand_parent: API
---

# Get a node's name

Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.

## General information about this method

> **Signature**: `=DCGETNAME(dcids)`

Reviewer: Should these be "Formula" instead for Sheets?

Author: Yep, that is a GREAT idea!

**Required arguments**:

* `dcids` - A list of nodes to query, identified by their DCIDs.

## Examples

Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).

> ### Example 1: Retrieve the common name of the country of Côte d'Ivoire.

Reviewer: Maybe leave "Cote d'Ivoire" out from the header here, so as not to confuse between this and "common" name
```suggestion
### Example 1: Retrieve the name of a country by it's DCID.
```

Author: This is a great idea! I've incorporated it.
